pkg/promunifi/uap.go

```diff
@@ -164,9 +164,11 @@ func descUAP(ns string) *uap {
 }
 
 func (u *promUnifi) exportUAP(r report, d *unifi.UAP) {
-	labels := []string{d.IP, d.Version, d.Model, d.Serial, d.Type, d.Mac, d.SiteName, d.Name}
+	labels := []string{d.Type, d.SiteName, d.Name}
+	infoLabels := []string{d.IP, d.Version, d.Model, d.Serial, d.Mac}
```

Comment:
Since we're doing this.... you can add a whole lot more labels than this.

Reply:
Looking at the data struct, I don't see a lot of additional things that make sense as part of this info label. Do you have any specific ones?

For example, I would probably suggest dropping the `IP` one from the info, as it's not really a device identifier.

Reply:
You're probably right, not much more needed here. This is what I reduced the tags to in influx: 
```
		"mac":       s.Mac,
		"site_name": s.SiteName,
		"name":      s.Name,
		"version":   s.Version,
		"model":     s.Model,
		"serial":    s.Serial,
		"type":      s.Type,
```
You may drop the IP from the devices too. It's only there for data tables, and with _this_ new metric we have it still.

Reply:
Yea, that looks like all the labels we currently have.